assets/js/util/insufficient-permissions-error-description.js

```diff
@@ -44,23 +44,23 @@ export function getInsufficientPermissionsErrorDescription( error, module ) {
 	if ( 'analytics' === slug ) {
 		if ( error.match( /account/i ) ) {
 			message = __(
-				`Your Google account does not have sufficient permissions for this Analytics account, so you won't be able to see stats from it on the Site Kit dashboard`,
+				"Your Google account does not have sufficient permissions for this Analytics account, so you won't be able to see stats from it on the Site Kit dashboard",
```

Comment:
Same here. Please, use the `’` symbol for the apostrophe and single quotes for the string. The same applies to all changes in this file.